tsdb/head.go

```diff
@@ -524,23 +531,49 @@ Outer:
 					seriesCreationErr = err
```

Comment:
Why not `series, created, seriesCreationErr := ` above? :thinking:

Reply:
To be explicit that the scope of `seriesCreationErr` is outside the loop, else its not straightforward to know if we use `:=`